internal/proxy/proxy.go

```diff
@@ -801,6 +807,8 @@ func (c *Client) newSocketMount(ctx context.Context, conf *Config, pc *portConfi
 
 		address, err = newUnixSocketMount(inst, conf.UnixSocket, strings.HasPrefix(version, "POSTGRES"))
 		if err != nil {
+			c.logger.Errorf("could not mount unix socket %q for %q: %v",
```

Comment:
Let's consolidate all logging up above in NewClient.

Reply:
I see that we're not doing that consistently above. We should confirm that we're not logging this error twice, though.

Reply:
As discussed offline, removed the NewClient logs and kept logs at the newSocketMount level.